src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorUtil.groovy

```diff
@@ -92,7 +92,7 @@ class IqPolicyEvaluatorUtil
       throw e
     }
     else {
-      listener.logger.println Messages.IqPolicyEvaluation_UnableToCommunicate(e.message)
+      e.printStackTrace(listener.error(e.getMessage()));
```

Comment:
if we switch to just printing the stack trace and we take out the listener, are there any side effects from it?  I'm not sure why the listener was put in place before.

Reply:
>f we switch to just printing the stack trace and we take out the listener, are there any side effects from it?

```e.printStackTrace(listener.error(e.getMessage()));``` not side efffects.  This is what jenkins does in their code.

Reply:
> if we switch to just printing the stack trace and we take out the listener, are there any side effects from it? I'm not sure why the listener was put in place before.

Changed a2aa47